src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/RuneOfUnemployment.java

```diff
@@ -0,0 +1,62 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.magical;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.bukkit.GameMode;
+import org.bukkit.Location;
+import org.bukkit.Particle;
+import org.bukkit.entity.Villager;
+import org.bukkit.entity.Villager.Profession;
+import org.bukkit.entity.WanderingTrader;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.Merchant;
+import io.github.thebusybiscuit.cscorelib2.inventory.ItemUtils;
+import io.github.thebusybiscuit.slimefun4.core.handlers.EntityInteractHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.Objects.handlers.ItemHandler;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+/**
+ * This {@link SlimefunItem} allows you to reset a {@link Villager} profession. 
+ * Useful to reset a villager who does not have desirable trades.
+ *
+ * @author dNiym
+ *
+ * @see ItemHandler
```

Comment:
Why would you reference "ItemHandler" here?

Reply:
I am not entirely sure, I based this off of the RandomMobDrop item that I had already done and it did use the ItemHandler, probably leftover from the NotPlacable reference.   Would a @see EntityInteractHandler be more appropriate?

Reply:
er or maybe I did it because of the -> getItemHandler() call

Reply:
EntityInteractHandler would probably be more appropriate.
Not sure if we really need any see tag here though in the first place.

Reply:
done